R/as.data.frame.R

```diff
@@ -157,7 +157,7 @@ as.data.frame.dcm <- function(x, row.names = NULL, optional = FALSE, run,
 #'
 #' \dontrun{
 #' ## Stochastic SI network model
-#' nw <- network.initialize(n = 100, directed = FALSE)
+#' nw <- network_initialize(n = 100, directed = FALSE)
```

Comment:
remove all the `directed = FALSE` throughout since that is now the default with network_initialize

Reply:
Done. Also cleaned up formatting so all function calls are consistent.

Reply:
@andsv2 as noted, *please* mark these done with "resolve conversation"